libs/model/src/models/reaction.ts

```diff
@@ -73,6 +74,21 @@ export default (sequelize: Sequelize.Sequelize) =>
                     transaction: options.transaction,
                   });
                 }
+                if (reaction.reaction === 'like') {
```

Comment:
let's discuss this approach - don't like coupling the data model with the responsibility of explicitly emit events, this should be part of the command handlers, where the emitEvent hook is just a generic sequelize utility.

Reply:
@Rotorsoft this is the pattern we agreed upon per https://github.com/hicommonwealth/commonwealth/issues/6939. I'm fine with emitting as part of command handlers but we don't have any command handlers for upvotes right now so we would be adding this logic to a legacy route. I think we should maintain the current pattern for now and then migrate to emitting from commands when we actually have commands.

Reply:
Right, I know we agreed upon this pattern... just making sure this is a big red flag in the models... approving under protest 😆